data/plugins/language_c.lua

```diff
@@ -111,18 +118,6 @@ syntax.add {
     ["true"]     = "literal",
     ["false"]    = "literal",
     ["NULL"]     = "literal",
-    ["#include"] = "keyword",
```

Comment:
The autocomplete plugin currently has a feature that reads a syntax symbols and offers to complete them while typing so having this symbols may be useful when a person does not have any sort of LSP enabled. The only drawback in this scenario is that modifying the `config.symbol_pattern` may be required to support `#` as part of words